src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -796,11 +538,319 @@ private void OnDebugModeFileChanged(object sender, FileSystemEventArgs e)
             LastDebugNotify = DateTime.UtcNow;
         }
 
-        private void BlobLeaseManagerHasLeaseChanged(object sender, EventArgs e)
+        private void OnHostLeaseChanged(object sender, EventArgs e)
         {
             IsPrimaryChanged?.Invoke(this, EventArgs.Empty);
         }
 
+        /// <summary>
+        /// Perform any early initialization operations.
+        /// </summary>
+        private void Preinitialize()
```

Comment:
Nit: should be pascal cased e.g. PreInitialize()

Reply:
I knew it looked weird but couldn't figure out why...